### PR TITLE
krew v0.4.1

### PR DESCRIPTION
**Pull request by ahmetb**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/krew.yaml` is not a straightforward version bump: change on line `0` does not seem like a version bump: [`-# Copyright 2019 The Kubernetes Authors.`]

**Comment by corneliusweig** [comment]
> F0217 21:28:12.781386    2852 main.go:62] spec.platform[1]'s selector (&LabelSelector{MatchLabels:map[string]string{arch: arm64,os: darwin,},MatchExpressions:[]LabelSelectorRequirement{},}) doesn't match any supported platforms

Is that fixed already?

/lgtm
/approve
/hold

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
         export GOBIN=$HOME/bin
         # TODO(corneliusweig): pin validator version after next release
         # go get sigs.k8s.io/krew/cmd/validate-krew-manifest@${KREW_VERSION}
-        go get sigs.k8s.io/krew/cmd/validate-krew-manifest@60ca400404fcd12c9f14b68aead7191ae4748e4d
+        go get sigs.k8s.io/krew/cmd/validate-krew-manifest@ec169f8c9c53e0aaaf57f1f73fb38590842ceb77
```
Comment (corneliusweig) [inline .github/workflows/ci.yml:49]:
```suggestion
        go get sigs.k8s.io/krew/cmd/validate-krew-manifest@v0.4.1
```

**Review by corneliusweig — COMMENTED** [review]
Suggestion optional for now, but would be cleaner at some point.

/lgtm

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ahmetb) [reply .github/workflows/ci.yml:49]: I suspect we'll  run into more hotfixes like that, so leaving as-is.

**Pull request merged**